Fix capitalization and hyphenation of DC2DC (#1164)
arangodb/docs: 3.11/arangosync.md

---
layout: default
description: >-

>   A detailed guide into deploying a datacenter-to-datacenter replicated environment

Author: ```suggestion
  A detailed guide in deploying datacenter-to-datacenter replicated clusters
```
